modules/nw-egress-ips-about.adoc

```diff
@@ -149,6 +149,37 @@ On Azure, the following capacity limits exist for IP address assignment:
+
+By using non-OVN managed networks, {product-title} administrators are provided with a greater level of control over networking aspects, such as routing, addressing, segmentation, and security policies. Non-OVN managed networks also provide users with the option to route workload traffic over specific network interface cards (NICs) for purposes such as traffic segmentation or meeting specialized requirements.
+
+For users who want to assign non-OVN managed networks to their egress IP addresses, the following conditions must be met:
+
+* {product-title} is installed on a bare metal cluster. Assigning non-OVN managed networks to egress IP addresses is disabled in cloud and hypervisor environments, and does not work for host-networked pods. 
+* A standard Linux network link is required. If a network link is removed or if the IP address on the link is removed, then egress IP is unwired and reassigned to another location. 
+* Egress IP addresses must have a universal, or _global_, scope. This means that in order to assign an egress IP address to a network link, the system examines all IP addresses with global scopes and their associated subnets. For example, if an egress IP falls within a specific `/24` subnet such as `10.0.0.0/24`, it can be assigned to the corresponding link. The scope of a link can be determined using standard Linux tools, and you also have the ability to set the scope of an address on a link using these tools. For more information about address scopes and values, see link:https://man7.org/linux/man-pages/man8/ip-address.8.html[Linux manual page] on IP addresses.
+* Links and their addresses must not be removed during runtime after an egress IP address is assigned to it. If you want to remove the link, you must first remove the egress IP address and then remove the address or network link. 
+* IP forwarding must be enabled for the network link. To enable IP forwarding, you can set `ipForwarding: Global` in your `config.yaml` file, for example:
++
+[source,yaml]
+----
+# ...
+ipForwarding: Global
+# ...
+----
+
+[discrete]
+[id="nw-egress-ips-multi-nic-limitations_{context}"]
+=== Limitations for assigning non-OVN managed networks to egress IP addresses
+
+The following limitations apply when assigning egress IP addresses to non-OVN managed networks:
+
+* Assigning non-OVN managed networks to egress IP addresses is unsupported if you have configured the {openshift-networking} OVN-Kubernetes network plugin to use additional networks.
+* Attempting to use `br-ex`, or external bridge, for egress IPs by manually adding additional IPs is unsupported. `br-ex` is an OVN-Kubernetes managed interface and only supports the primary OVN-Kubernetes network.
```

Comment:
You can remove this line and just add:

```suggestion
* Only network interfaces of type bond and VLAN are supported.
* Only IP version 4 is supported.
```

Reply:
@martinkennelly yeah for now we'll just say "Only IP v4 is supported" and when the bug is fixed we can make the changes. I'll just need to know when the bug is fixed.